domains/arman.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Arman's personal website!",
+  "repo": "https://github.com/ItzArman09/website",
+  "owner": {
+    "username": "ItzArman09",
+    "email": "gyfufifa@norwegischlernen.info"
+  },
+  "record": {
+    "CNAME": "arman.is-a.dev"
```

Comment:
this should be points to your host, not your domain itself.

see your host's custom domain instruction for more info.

Reply:
uhm, im using github pages

Reply:
edited, check now

Reply:
![image](https://user-images.githubusercontent.com/100463136/161045427-b4cfe2fd-858b-42a2-8388-8efe438f5c0e.png)
whats this?

Reply:
the pr didnt get merged

a maintainer would need to merge before dns record get added

Reply:
okay thank you